modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/paths/PRMAnnouncements.path

```diff
@@ -0,0 +1,21 @@
+<html>
+<head>
+<title>PRMAnnouncements</title>
+</head>
+
+<body>
+<table border="1" cellpadding="1" cellspacing="1">
+<thead>
+<tr><td rowspan="1" colspan="3">PRMAnnouncements</td></tr>
+</thead>
+
+<tbody>
+<tr>
+	<td>ANNOUNCEMENT_TITLE_AND_CONTENT</td>
+	<td>//div[contains(@class,'border-neutral')]//*[contains(text(),'${key_announcementTitle}')]/following-sibling::div//p[text()='${key_announcementContent}']</td>
+	<td></td>
```

Comment:
Hi @marinaasant0s, could you please change this path to `//div[@data-analytics-asset-title='${key_announcementTitle}']/*[contains(.,'${key_announcementContent}')]`? I think that on this way will be more consistent and less fragile